server/status/recorder.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 const (
-	storeTimeSeriesNameFmt = "cr.store.%d.%s"
+	storeTimeSeriesNameFmt = "cr.store.%s.%d"
```

Comment:
Went ahead and swapped the name and store number for the purpose of efficient queries. The query system is going to fill in wildcards with simple scans over a sorted list of metric keys, so it will be best to have our most common wildcards be suffixes.